package.json

```diff
@@ -14,90 +14,41 @@
   },
   "homepage": "https://github.com/formidablelabs/victory-line",
   "scripts": {
-    "postinstall": "node -e \"require('fs').stat('lib', function(e,s){process.exit(e || !s.isDirectory() ? 1 : 0)})\" || npm run build-lib",
-    "preversion": "npm run check",
-    "version": "npm run clean && npm run build && git add -A dist",
-    "clean-dist": "rimraf dist",
-    "build-dist-min": "webpack --config webpack.config.js",
-    "build-dist-dev": "webpack --config webpack.config.dev.js",
-    "build-dist": "npm run clean-dist && npm run build-dist-min && npm run build-dist-dev",
-    "clean-lib": "rimraf lib",
-    "build-lib": "npm run clean-lib && babel --stage 0 src -d lib",
-    "clean": "npm run clean-lib && npm run clean-dist",
-    "build": "npm run build-lib && npm run build-dist",
-    "server-dev": "webpack-dev-server --port 3000 --config demo/webpack.config.dev.js --content-base demo",
-    "server-hot": "webpack-dev-server --port 3000 --config demo/webpack.config.hot.js --hot --content-base demo",
-    "server-test": "webpack-dev-server --port 3001 --config webpack.config.test.js",
-    "dev": "npm run server-dev & npm run server-test",
-    "hot": "npm run server-hot & npm run server-test",
+    "postinstall": "builder run npm:postinstall",
+    "preversion": "builder run npm:preversion",
+    "version": "builder run npm:version",
+    "test": "builder run npm:test",
     "build-docs": "webpack --config docs/webpack.config.dist.js",
     "deploy-docs": "npm run generate-docs & npm run build-docs & git subtree push --prefix docs/dist origin gh-pages",
     "dev-docs": "npm run generate-docs & webpack-dev-server --port 3000 --config docs/webpack.config.dev.js --content-base docs",
     "generate-docs": "node_modules/react-docgen/bin/react-docgen.js src/components/victory-line.jsx -o docs/victory-line.json",
     "hot-docs": "npm run generate-docs & webpack-dev-server --port 3000 --config docs/webpack.config.hot.js --hot --content-base docs",
```

Comment:
Didn't break in this PR, but might as well catch it: if you run `hot-docs` you might notice it's serving up the axis demo instead of the docs. `docs/webpack.config.hot.js` needs to be fixed from loading `"./demo/app.jsx"` to loading `"./docs/docs.jsx"`.